vim-tmux-navigator.tmux

```diff
@@ -22,7 +22,7 @@ get_tmux_option() {
 }
 
 # Export 'vim_pattern' so that it can be tested in pattern-check
-declare vim_pattern='(\\S+\\/)?g?\.?(view|l?n?vim?x?|fzf)(diff)?(-wrapped)?$'
+declare vim_pattern='(\\S+/)?(^|/)g?\.?(view|l?n?vim?x?|fzf)(diff)?(-wrapped)?$'
```

Comment:
Let me know how you feel about trimming out the extra `'\'` characters. I can add them back in (doesn't seem to hurt). If I add it back in, then this line will look like:

```diff
- declare vim_pattern='(\\S+/)?(^|/)g?\.?(view|l?n?vim?x?|fzf)(diff)?(-wrapped)?$'
+ declare vim_pattern='(\\S+\\/)?(^|\/)g?\.?(view|l?n?vim?x?|fzf)(diff)?(-wrapped)?$'
```